nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSRepositoryDescriptor.java

```diff
@@ -95,6 +95,54 @@ public void setFulltextIncludedTypes(Set<String> fulltextIncludedTypes) {
         fulltextDescriptor.setFulltextIncludedTypes(fulltextIncludedTypes);
     }
 
+    /** @since 8.10 */
+    @XNode("cache@enabled")
+    private Boolean cacheEnabled;
+
+    /** @since 8.10 */
+    public boolean isCacheEnabled() {
+        return defaultFalse(cacheEnabled);
+    }
+
+    /** @since 8.10 */
+    protected void setCacheEnabled(boolean enabled) {
+        cacheEnabled = Boolean.valueOf(enabled);
+    }
+
+    /** @since 8.10 */
+    @XNode("cache@ttl")
+    public Long cacheTtl;
+
+    /** @since 8.10 */
+    @XNode("cache@maxSize")
+    public Long cacheMaxSize;
+
+    /** @since 8.10 */
+    @XNode("cache@concurrencyLevel")
```

Comment:
Indeed it's a guava cache property, I can add some description from Guava javadoc if you want.